go.mod

```diff
@@ -7,6 +7,7 @@ require (
 	github.com/bsipos/thist v1.0.0
 	github.com/btcsuite/btcd v0.21.0-beta
 	github.com/davecgh/go-spew v1.1.1
+	github.com/dgraph-io/badger v1.6.1
```

Comment:
We should be using `v2`, not sure why `v1.x` is being added here

Reply:
Oh that's really strange, I'll remove that. Might have got auto added as I was writing the tests.